GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -153,6 +154,7 @@ @interface GTViewController () <MFMailComposeViewControllerDelegate, UIActionShe
 @property (nonatomic, assign)	BOOL isRotated;
 @property (nonatomic, assign)	BOOL instructionsAreRunning;
 
+@property (nonatomic, weak)     GTFollowupViewController    *followupViews;
```

Comment:
this should probably be named followupView instead of followupViews. the s suggests it is an array of views.